basics.tex

```diff
@@ -109,10 +109,12 @@ \section{Basic Data Types}\label{s:basics-data-types}
 the type of null is object
 \end{minted}
 
-As the example above shows,
-we use \texttt{let} to define a \gref{g:variable}{variable},
-put values separated by commas inside \texttt{{[}{]}} to create an \gref{g:array}{array},
-and use \texttt{for...of} to loop over the values in that array.
+As the example above shows, we create an array of values to be looped through called 
+\texttt{otherValues}. We initiate our for loop with the word \texttt{for}. Within the parentheses, 
+\texttt{let} creates a loop to iterate over each element within \texttt{otherValues}, and \texttt{value}.  
```

Comment:
1. word missing: "let create a loop _variable_ to iterate..."
2. grammar: I think the period at the end of this line is misplaced (the next line seems to be a continuation of the same sentence).

So maybe "let creates a variable called value to iterate over each element within otherValues".